src/test/java/cucumber/hooks/Hooks.java

```diff
@@ -52,12 +52,12 @@ public void generateToken() {
     public void setUp() {
         requestBuilder
                 .addHeader("Authorization", token)
-                .addBaseUri(getTheAdminUrl());
+                .addBaseUri("https://jalasft2-dev-ed.my.salesforce.com/");
```

Comment:
you should receive this value from an environment file and not put it hardcoded.